### DISCUSSION TITLE
Launching second Winforms CefSharp No Longer Works

### DISCUSSION THREAD

**Discussion by bens613** [cefsharp/CefSharp#4891] [Q&A]
I upgraded CefSharp from an older version and now I cannot launch my WinForms application more than once (from the same directory). It used to work fine and I could even click debug a few times in Visual Studio. Now the first one launches my application but the second one launches a Chrome window with a warning about sandbox invalid input parameter?

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/issues/4668#issuecomment-1890247564

The chromium process Singleton now only allows access to a single process

**Reply by bens613** [reply]
What does this mean? So if I want to launch multiple instances it should be done in tabs or something else?

**Reply by bens613** [reply]
https://github.com/cefsharp/CefSharp/issues/4668

**Comment by bens613** [comment]
See https://github.com/cefsharp/CefSharp/issues/4668.
